src/Constants/WebhookEvents.php

```diff
@@ -1,28 +1,13 @@
 <?php
+
+namespace UnzerSDK\Constants;
```

Comment:
I moved th phpDoc to the class so that it counts for the class and not for the file. That way the description e.g. also is shown when hovering over a class in the IDE. Did it mainly for classes that a merchant/developer might use.